src/main/java/aj/Ui.java

```diff
@@ -0,0 +1,118 @@
+package aj;
+
+/**
+ * Ui class responsible for User Interface of the programme.
+ */
+public class Ui {
+    TaskList taskList;
+
+    public void horiLine() {
+        System.out.println("---------------------");
+    }
```

Comment:
Perhaps you could make horiLine as a constant variable instead, and also use a better naming like horizontalLine instead?